service/routes.py

```diff
@@ -50,7 +50,7 @@ def check_price(price):
 
 def check_rating(rating_str):
     rating = float(rating_str)
-    if rating <= 0 or rating > 5:
+    if rating < 1 or rating > 5:
```

Comment:
In model.py, we checked values for `deserialize` function, no checking for `list_by_*`. If we don't add checking in `routes.py`, we might query strange things direct to our database. Thus I don't think we can simplify here.

Reply:
I see your point. It is not necessary for list_all, but it's needed for list_by_* since * is given by the user/sent from the request. The Comment is for LIST_ALL thus a bit confusing. But clear now. Thanks